docs/users.rst

```diff
@@ -182,6 +182,26 @@ TODO
 
 .. _content_sets.yml:
 
+image_build_method
```

Comment:
Should we add this to the example above (L92-115)?

Reply:
Reckon so :) :+1: